### ISSUE TITLE
Failed opening Amazon.com website

### ISSUE THREAD

**Issue by bromoapp** [dzikoysk/Pandomium#14]
Hi, sorry if I report this issue here.
I want to use your project for creating a web scraping (I don't know would it a success or not)
But I succeeded in opening AliExpress, Google search, and even Facebook. But I got ERR_ABORTED every time I try to open Amazon site. Any clue why this happens?

**Comment by bromoapp** [comment]
Hi... sorry, but I found out its a CERT ERROR .... any clue how to resolve this?

**Comment by dzikoysk** [comment]
Yeah, I'm not able to visit `https://www.amazon.com/` too:

```
[0210/141929.435:ERROR:child_thread_impl.cc(762)] Request for unknown Channel-associated interface: ui::mojom::GpuMain
[0210/141929.436:ERROR:gpu_child_thread.cc(330)] Exiting GPU process due to errors during initialization
```

JCEF Tracker can be much more useful: https://bitbucket.org/chromiumembedded/java-cef/issues?status=new&status=open

I've just found this: https://github.com/cztomczak/cefpython/issues/353
It looks similar to this, maybe I should update natives.

**Comment by dzikoysk** [comment]
Fixed in 67.0

![Amazon.com CEF Preview](https://user-images.githubusercontent.com/4235722/42050691-ecb3df52-7b09-11e8-83e7-32207ae164af.png)
